core/internal/testutils/configtest/general_config.go

```diff
@@ -46,6 +47,7 @@ type GeneralConfigOverrides struct {
 	Dialect                                   dialects.DialectName
 	EVMDisabled                               null.Bool
 	EthereumDisabled                          null.Bool
+	EthereumURL                               null.String
```

Comment:
Why was this added?

Reply:
I added as part of the overrides cause i wanted to add the ability to manually set the ETH URL manually, instead of depending on whatever value was being set behind the scenes. I could delete it if fine not having this as part of the overrides.